src/autocomplete/docs/index.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import Box from 'ui-box'
+import starWarsNames from 'starwars-names'
+import Autocomplete from '../src/Autocomplete'
+import { TextInput } from '../../text-input'
+import { Button } from '../../buttons'
+
+// Import main component as 'raw' text to be parsed with an AST to get get propTypes information
+/* eslint-disable import/no-unresolved, import/no-webpack-loader-syntax */
+import autocompleteSource from '!raw-loader!../src/Autocomplete'
+/* eslint-enable import/no-unresolved, import/no-webpack-loader-syntax */
+
+/**
+ * Code examples
+ */
+import exampleDefaultBasic from './examples/default-basic.example'
+import exampleDefaultWithProp from './examples/default-with-children-prop.example'
+import exampleonFocusBasic from './examples/onFocus-basic.example'
+import examplefilterDisabledBasic from './examples/filterDisabled-basic.example'
+import examplewithTitleBasic from './examples/withTitle-basic.example'
+import exampleWithButtonBasic from './examples/withButton-basic.example'
+import exampleWithManyOptions from './examples/allOptions.example'
+
+// Generate a big list of items
+const items = [
+  ...starWarsNames.all,
+  ...starWarsNames.all.map(x => `${x} 2`),
+  ...starWarsNames.all.map(x => `${x} 3`)
+].sort((a, b) => {
+  const nameA = a.toUpperCase()
+  const nameB = b.toUpperCase()
+  if (nameA < nameB) {
+    return -1
+  }
+  if (nameA > nameB) {
+    return 1
+  }
+  return 0
+})
+
+const title = 'Autocomplete'
+const subTitle = 'A component to filter trhough a dataset'
+
+const introduction = (
```

Comment:
introduction => implementationDetails.

introductions should try to use accessible language to both designers and engineers on the long run.

Reply:
We can ignore the introduction and just use implementationDetails instead.